### ISSUE TITLE
[Improvement] [xApp] Add xApp command to open account selector

### ISSUE THREAD

**Issue by WietseWind** [XRPL-Labs/XUMM-Issue-Tracker#333]
Just like the QR scanner, an xApp would benefit greatly from having access to an "recipient selector" for selecting a destination. We can re-use the "Send" flow destination selector, so own accounts, contacts, entering an r-address, resolving bithomp name, PayString, scanning a QR code with r-address are all handled.

This is what the flow should look like:
1. The developer issues a command from within the xApp, as with the QR scanner flow: `{ "command": "scanQr" }`, but now: 
```
{
  "command": "selectDestination"
}
```
2. The `Recipient selector` screen opens: https://user-images.githubusercontent.com/4756161/116758329-1563e900-aa10-11eb-888b-657a3c76b1e7.png - in this screne the *Back* button is replaced with Close, and the *Next* button can stay as is. Destination tag check + entering would stay the same. When an account is selected (and in case a contact with tag, or exchange account detected and tag entered) this screen resolves.
3. The xApp now receives a PostMessage:
```
{
  "method": "selectDestination",
  "destination": null,
  "info": null,
  "reason": "USER_CLOSE"
}
```
or in case of a selected account:
```
{
  "method": "selectDestination",
  "destination": {
    "address": "r...",
    "tag": 123456,
    "name": "Wietse's Savings"
  },
  "info":{
   "blackHole":false,
   "disallowIncomingXRP":false,
   "exist":true,
   "possibleExchange":false,
   "requireDestinationTag":false,
   "risk":"UNKNOWN"
  },
  "reason": "SELECTED"
}
```

The `account.name` field contains:
- In case of a contact: the contact name
- In case of a XUMM managed XRPL account: the account label
- In case of an account with resolved name (through Bithomp etc.): the name
- Else: null

**Comment by WietseWind** [comment]
@nixer89 You'll lovet his as well, for eg. the escrow destination picker.

**Comment by WietseWind** [comment]
Now available in the released XUMM 2.1.1
